Exercise_3.java

```diff
@@ -1,6 +1,6 @@
 import java.io.*; 
-//Time Complexity :  O(1)
-// Space Complexity :   O(n)
+//Time Complexity :  O(1) for each operation

```

Comment:
TC will O(n) for both insert and print operations because you are iterating through the whole list